packages/fractor-xml/src/Printer/PrettyXMLPrinter.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorXml\Printer;
+
+/**
+ * @source https://github.com/shanethehat/pretty-xml/blob/master/src/PrettyXml/Formatter.php
+ */
+class PrettyXMLPrinter
```

Comment:
We can require the library directly IMHO. I would create a custom interface for that and wrap it around to be more flexible.

Reply:
I require the lib directly now but I'm not sure how it would look like with the interface. Feel free to push some improvments.

Reply:
Additionally the e2e tests are failing now because of some DI issue wich I don't know how to solve.

Reply:
Fixed it